lib/bolt/analytics.rb

```diff
@@ -139,6 +141,15 @@ def report_bundled_content(mode, name)
         end
       end
 
+      def plan_counts(project_path)
+        pp_count, yaml_count = %w[pp yaml].map do |extension|
+          Find.find(File.join(project_path, 'plans')).grep(/.*\.#{extension}/).length
+        end
+
+        # Hash.new(%w[puppet_plan_count yaml_plan_count].zip(pp_count, yaml_count))
```

Comment:
```suggestion
```

I'm guessing this is leftover?

Reply:
Mostly to deter comments on how to calculate it, since it's more characters and operations :P I'll remove!